diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt
index 05a549aa5c7f..75e761703661 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt
@@ -1116,6 +1116,15 @@ class HomeFragment : Fragment(), UserInteractionHandler, OnLongPressedListener {
     override fun onResume() {
         super.onResume()
 
+        org.mozilla.fenix.components.SymfonyBuiltInExtensions.dashboardUrl?.let { dashboard ->
+            (requireActivity() as HomeActivity).openToBrowserAndLoad(
+                searchTermOrURL = dashboard,
+                newTab = true,
+                from = BrowserDirection.FromHome,
+            )
+            return
+        }
+
         hideToolbar()
 
         val components = requireComponents
